plugin/plugins/filesystem/client/client.go

```diff
@@ -0,0 +1,168 @@
+package p9client
```

Comment:
This is intended to become a wrapper library for clients to use. So that they can dial our plugin's server easily. Longer term it should shape the VFS interface. Currently we just use 9P semantics, but we can define operations such as `Readdir` here to be more brief, much like how MFS does with operations like `Mkdir`.

e.g.
doing `ents, err := client.Readdir(someFilesystem, "/some/path")` rather than using the interface directly yourself.

However this is more related to https://github.com/ipfs/package-managers/issues/71 and should be broken up. This PR should only relate to the server portion at the moment so I'll remove this from the patchset.